docs/plugins/repository-azure.asciidoc

```diff
@@ -19,7 +19,8 @@ bin/elasticsearch-keystore add azure.client.default.account
 bin/elasticsearch-keystore add azure.client.default.key
 ----------------------------------------------------------------
 
-Where `account` is the azure account name and `key` the azure secret key.
+Where `account` is the azure account name and `key` the azure secret key. Instead of an azure secret key under `key`, you can alternatively
+define a shared access signatures (SAS) token under `sas_token` to use for authentication instead.
```

Comment:
I think we should say a word about the type of SAS token and the required permissions

Reply:
I think the type doesn't matter just the permissions (service as well as accounts SAS should work)? I added a comment on the permissions now :)

Reply:
Talked to Armin using another channel. I'd expect to see some `create` or `list` permissions too here

Reply:
Alright, updated this with all the details now. `create` we don't need since it's implied by `write` for the operations we execute https://docs.microsoft.com/en-us/rest/api/storageservices/Constructing-an-Account-SAS?redirectedfrom=MSDN#blob-service (as a matter of fact it is for all operations on the blob service). But we needed `list`.
Also, I added some detail on the service and resource type the token has to apply to.